---
title: Gene co-expression networks highlight key nodes associated with Ammonium nitrate
  in sugarcane

# Authors
# A YAML list of author names
# If you created a profile for a user (e.g. the default `admin` user at `content/authors/admin/`), 
# write the username (folder name) here, and it will be replaced with their full name and linked to their profile.
authors:
- Jorge Mario Munoz-Perez
- Denis Bassi
- Lucia Mattiello
- Marcelo Menossi
- Diego M Riano-Pachon

# Author notes (such as 'Equal Contribution')
# A YAML list of notes for each author in the above `authors` list
author_notes: []

date: '2025-01-01'

# Date to publish webpage (NOT necessarily Bibtex publication's date).
publishDate: '2025-05-16T13:45:31.420140Z'

# Publication type.
# A single CSL publication type but formatted as a YAML list (for Hugo requirements).
publication_types:
- '2'

# Publication name and optional abbreviated publication name.
publication: '*bioRxiv*'
publication_short: ''

doi: 10.1101/2025.05.14.652917

abstract: Gene co-expression network analysis offers valuable insights into complex
  biological processes in plants. In this study, we investigated the molecular mechanisms
  underlying nitrogen utilization efficiency (NUE) in sugarcane by analyzing transcriptome
  data from two contrasting genotypes (responsive and non-responsive) across different
  leaf segments under varying nitrogen conditions. RNA-seq analysis of 48 samples
  identified 2,723 differentially expressed genes. The responsive genotype showed
  enrichment in carbon metabolism and defense response pathways under high nitrogen
  conditions, while the non-responsive genotype prioritized photosynthesis and stress
  response mechanisms. Differentially expressed genes were used to construct a co-expression
  network which revealed 199 distinct modules. 44 modules correlating to nitrogen
  availability and 20 to genotype. Notably, Module 20, containing a high proportion
  of MYB/MYB-related transcription factors, emerged as a potential key regulator of
  nitrogen response in both genotypes. Several modules showed correlation with important
  metabolic markers such as RUBISCO, chlorophyll, and PEPCASE. Additionally, we identified
  novel candidate genes, including uncharacterized transcripts with strong genotype-specific
  expression patterns. These findings provide potential targets for understanding
  and improving nitrogen use efficiency in sugarcane breeding programs.Competing Interest
  StatementThe authors have declared no competing interest.Conselho Nacional de Desenvolvimento
  Cientı́fico e Tecnológico, Brasilia, BR, 311558/2021-6, 442529/2014-7Fundação de
  Amparo à Pesquisa do Estado de São Paulo, 2013/15576-5, 2012/23345-0Coordenação
  de Aperfeicoamento de Pessoal de Nv́el Superior, Finance Code 001

# Summary. An optional shortened abstract.
summary: ''

tags: []

# Display this page in a list of Featured pages?
featured: false

# Links
url_pdf: ''
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Custom links (uncomment lines below)
# links:
# - name: Custom Link
#   url: http://example.org

# Publication image
# Add an image named `featured.jpg/png` to your page's folder then add a caption below.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects: ['internal-project']` links to `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []
links:
- name: URL
  url: https://www.biorxiv.org/content/early/2025/05/16/2025.05.14.652917
---

