---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Gene Co-Expression Networks Highlight Key Nodes Associated With Ammonium Nitrate
  in Sugarcane.
subtitle: ''
summary: ''
authors:
- Jorge Mario Muñoz-Pérez
- Denis Bassi
- Lucia Mattiello
- Marcelo Menossi
- Diego Mauricio Riaño-Pachón
tags:
- Saccharum
- genetics
- metabolism; Gene Expression Regulation
- Plant; Gene Regulatory Networks; Nitrates
- metabolism; Nitrogen
- metabolism; Transcriptome
- genetics; Plant Proteins
- genetics
- metabolism; Plant Leaves
- genetics
- metabolism; Genotype; Transcription Factors
- metabolism
- genetics; Gene Expression Profiling; Photosynthesis
- genetics; MYB transcription factors; gene co‐expression network; nitrogen use efficiency
  (NUE); sugarcane; transcriptomics
categories: []
date: '2025-01-01'
lastmod: 2025-11-14T14:05:03-03:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2025-11-14T17:05:03.499624Z'
publication_types:
- '2'
abstract: This study aimed to investigate molecular mechanisms underlying nitrogen
  use efficiency (NUE) in sugarcane by analyzing transcriptome profiles of two genotypes
  (responsive, RB975375; non-responsive, RB937570) under contrasting nitrogen conditions,
  and along the leaf development gradient. Using RNA-seq data from 48 leaf segment
  samples, the analyses integrated gene co-expression network approaches to uncover
  genotype-specific regulatory modules and metabolic pathways linked to NUE. The responsive
  genotype prioritized carbon metabolism and defense under high nitrogen, while the
  non-responsive genotype activated photosynthesis and stress responses. Co-expression
  analysis revealed 44 nitrogen-responsive and 20 genotype-correlated modules. Module
  20, enriched in MYB/MYB-related transcription factors, emerged as a central regulator
  of nitrogen response. Key metabolic markers (RUBISCO, PEPCASE) correlated with specific
  modules, and novel candidate genes (e.g., NewTr2475430.gen) showed genotype-specific
  expression. Generated resources include (1) RNA-seq datasets (NCBI BioProject PRJNA1176579);
  (2) a de novo transcriptome assembly (3.8 million transcripts clustered into 2.48 million
  transcript groups); (3) co-expression networks (1109 nodes, 199 modules); (4) annotated
  DEGs (2723) and metabolic correlations (e.g., RUBISCO, chlorophyll); (5) genotype-specific
  expression profiles and candidate genes (e.g., MYB transcription factors, uncharacterized
  transcripts). This resource provides actionable targets (e.g., MYB TFs, uncharacterized
  transcripts) for improving NUE in sugarcane breeding. The network modules and metabolic
  correlations offer a systems-level framework to study complex nitrogen-responsive
  mechanisms in polyploid C4 crops. Publicly available datasets enable comparative
  studies on nutrient use efficiency in polyploid crops, advancing sustainable agriculture.
publication: '*Physiologia plantarum*'
doi: 10.1111/ppl.70612
links:
- name: URL
  url: https://doi.org/10.1111/ppl.70612
---
