---
# Documentation: https://wowchemy.com/docs/managing-content/

title: "Tools"
subtitle: ""
summary: ""
authors: []
tags: []
categories: []
date: 2023-11-16T23:04:24-02:00
lastmod: 2023-11-16T23:04:24-02:00
featured: false
draft: false
comments: false  
# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ""
  focal_point: ""
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
---

# Blast Server

We have a public blast server that provides access to some of the datasets generated by members of the group. [There](http://labbces.cena.usp.br:4567/) you can find our current polyploid genome assembly for SP80-3280 (assembly B2) and a version of a pan-transcriptome assembly (longest transcript of group, Scp.v1) derived from 50 sugarcane genotypes.

# Digital Northern

We took the 20 year-old data generated as part of the SUCEST project, and made a new minning application of gene expression values per library, available [here](https://bces.shinyapps.io/ESTs_App/), the EST libraries shown there are described in this [paper](https://pubmed.ncbi.nlm.nih.gov/14613979/). We created this app, because this dataset is the only one available that includes callus.

We also created a [new app](https://bces.shinyapps.io/B2quant_App/), with expression values of genes annoted in our new genome assembly of cultivar SP80-3280. This will be available for the time being until we have the final version of the assembly and annotation adn move everything into Conekt Grasses.

# CoCoView

Are you interested in visualizing codon conservation from multiple sequences alignments. The traditional implementation of SequenceLogos cannot do that, this is why we developed CoCoView, check our [tool](https://github.com/labbces/CoCoView) and its companion [paper](https://pubmed.ncbi.nlm.nih.gov/35990812/).

# PloidyNGS

You have a reference genome, some short read genomics data, and whant to get a feeling of the ploidy of you object of study, say no more, use our tool [PloidyNGS](https://github.com/diriano/ploidyNGS), check out also its companion [paper](https://pubmed.ncbi.nlm.nih.gov/28383704/)
